frontend/src/components/NavBar/PrintImage/printConfig.tsx

```diff
@@ -94,11 +92,7 @@ function ToggleSelector({
             }}
             disabled={x.disabled}
           >
-            {typeof x.comp === 'function' && typeof iconProp === 'number' ? (
-              <x.comp value={iconProp} />
-            ) : (
-              x.comp
-            )}
+            {x.comp}
```

Comment:
Let's create a list of things to QA and add the print config to make sure we QA this properly